raiden/api/rest.py

```diff
@@ -525,6 +528,39 @@ def get_our_address(self) -> Response:
     def get_raiden_version(cls) -> Response:
         return api_response(result=dict(version=get_system_spec()["raiden"]))
 
+    def get_contract_versions(self) -> Response:
+        contracts = dict(
+            contracts_version=CONTRACTS_VERSION,
```

Comment:
here it would make a bit more of sense to use `self.raiden_api.raiden.proxy_manager.contract_manager.contracts_version` a very short path 😅